torch/nn/quantized/modules/conv.py

```diff
@@ -17,7 +17,9 @@
 
 _SUPPORTED_PADDING = {
     'zeros',
-    'reflect'
+    'reflect',
+    # 'circular',
```

Comment:
nit: is this needed?

Reply:
Nope -- was testing  the support. Will remove it